pkg/egressgateway/manager_privileged_test.go

```diff
@@ -541,10 +543,18 @@ func assertIPRules(c *C, rules []ipRule) {
 nextRule:
 	for _, rule := range parsedRules {
 		for _, installedRule := range installedRules {
-			if rule.sourceIP.Equal(installedRule.Src.IP) && rule.destCIDR.String() == installedRule.Dst.String() &&
+			ruleSrcIP, ok := ip.AddrFromIP(installedRule.Src.IP)
+			if !ok {
+				continue nextRule
+			}
```

Comment:
I guess this is happening because of the `route` pkg...unfortunate, but probably best left for another PR. Could you do the conversion of `installedRules` before the for-loop so as to remove the extraneous error handling in the loop?